src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -2422,6 +2435,9 @@ void M()
             using var md1 = diff1.GetMetadata();
             var reader1 = md1.Reader;
             var readers = new[] { reader0, reader1 };
+
+            diff1.VerifyUpdatedTypes("0x02000003");
```

Comment:
>0x02000003 [](http://example.com/codeflow?start=38&length=10)

I wonder if we could pass the `readers` to VerifyUpdatedTypes and resolve the tokens to type names for better readability of the tests. Something to perhaps follow up on.